rocket_sqlite_oracle/Rocket.toml

```diff
@@ -0,0 +1,2 @@
+[default.databases.example_signatures]
+url = "./db/oracle_signatures.sqlite"
```

Comment:
Same here, this binary is an oracle, I don't think the additional "oracle_" adds much value.